### ISSUE TITLE
Interest in including a "helper" function?

### ISSUE THREAD

**Issue by Dasonk** [DavidASmith/wordler#9]
I was just considering writing a package to play wordle in R and also I wanted to include some functions to 'help' people with potential solutions based on potential inputs.  Since you already have this package - would there be any interest in incorporating the helper type functions if I wrote them and submitted a pull request?

**Comment by DavidASmith** [comment]
Yes - I would be interested in incorporating these functions. Are you essentially considering writing a solver for the game?
